src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementCheckbox.tsx

```diff
@@ -1,59 +1,72 @@
+import { ColonyRole, ROOT_DOMAIN_ID } from '@colony/colony-js';
 
 import { Checkbox } from '~core/Fields';
-import Heading from '~core/Heading';
 import Popover from '~core/Popover';
+import PermissionsLabel from '~core/PermissionsLabel';
 
-import styles from './PermissionCheckbox.css';
+import styles from './PermissionManagementCheckbox.css';
+
+const idBase =
+  'dashboard.PermissionManagementDialog.PermissionManagementCheckbox';
 
 const MSG = defineMessages({
   roleWithAsterisk: {
-    id: 'admin.ColonyPermissionEditDialog.roleWithAsterisk',
+    id: `${idBase}.roleWithAsterisk`,
     defaultMessage: '{role}{asterisk, select, true {*} false {} }',
   },
   roleDescription0: {
-    id: 'admin.ColonyPermissionEditDialog.roleDescriptionRecovery',
-    defaultMessage: 'Put the Colony into recovery mode.',
+    id: `${idBase}.roleDescriptionRecovery`,
+    defaultMessage:
+      'Disable colony in emergency, update storage, and approvee reactivation',
   },
   roleDescription1: {
-    id: 'admin.ColonyPermissionEditDialog.roleDescriptionRoot',
-    defaultMessage:
-      'The highest permission, control all aspects of running a colony.',
+    id: `${idBase}.roleDescriptionRoot`,
+    defaultMessage: 'Take actions effecting the colony as a whole.',
   },
   roleDescription2: {
-    id: 'admin.ColonyPermissionEditDialog.roleDescriptionArbitration',
+    id: `${idBase}.roleDescriptionArbitration`,
     defaultMessage: 'Coming soon...',
   },
   roleDescription3: {
-    id: 'admin.ColonyPermissionEditDialog.roleDescriptionArchitecture',
-    defaultMessage: `Set the administration, funding, and architecture roles in any subdomain.`,
+    id: `${idBase}.roleDescriptionArchitecture`,
+    defaultMessage: `Set permissions in the active domain, and any subdomain.`,
   },
   // We don't have architecture_subdomain (which would be 4)
   roleDescription5: {
-    id: 'admin.ColonyPermissionEditDialog.roleDescriptionFunding',
-    defaultMessage: 'Fund tasks and transfer funds between domains.',
+    id: `${idBase}.roleDescriptionFunding`,
+    defaultMessage: 'Fund expenditures and transfer funds between domains.',
   },
   roleDescription6: {
-    id: 'admin.ColonyPermissionEditDialog.roleDescriptionAdministration',
-    defaultMessage: 'Create and manage new tasks.',
+    id: `${idBase}.roleDescriptionAdministration`,
+    defaultMessage: 'Create and manage expenditures.',
   },
   tooltipNoPermissionsText: {
-    id: 'admin.Permissions.PermissionCheckbox.tooltipNoPermissionsText',
+    id: `${idBase}.tooltipNoPermissionsText`,
     defaultMessage: 'You do not have permission to set the {roleName} role.',
   },
+  tooltipNoRootDomainSelected: {
+    id: `${idBase}.tooltipNoRootDomainSelected`,
+    defaultMessage: 'Switch domain to #Root to set the root role.',
+  },
 });
 
 interface Props {
   asterisk: boolean;
   disabled: boolean;
   role: ColonyRole;
+  domainId: number;
 }
 
 const displayName = 'admin.Permissions.PermissionCheckbox';
```

Comment:
This is no longer in the `admin` module since you moved the component. Please update the `displayName` to reflect that